src/main/java/mate/academy/spring/controller/UserController.java

```diff
@@ -0,0 +1,52 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.stream.Collectors;
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.UserResponseDto;
+import mate.academy.spring.service.UserService;
+import mate.academy.spring.service.mapper.UserDtoMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/users")
+public class UserController {
+    private final UserService userService;
+    private final UserDtoMapper userDtoMapper;
+
+    @Autowired
+    public UserController(UserService userService, UserDtoMapper userDtoMapper) {
+        this.userService = userService;
+        this.userDtoMapper = userDtoMapper;
+    }
+
+    @GetMapping("/inject")
+    public String injectData() {
+        userService.add(new User("Trinity","Human"));
+        userService.add(new User("Neo","Human"));
+        userService.add(new User("Smith","Agent"));
+        return "Injected users!";
+    }
+
+    @GetMapping("/{userId}")
+    public UserResponseDto get(@PathVariable Long userId) {
+        try {
+            return userDtoMapper.parse(userService.get(userId));
+        } catch (NoSuchElementException e) {
+            throw new RuntimeException("No user with id: " + userId);
+        }
```

Comment:
```suggestion
            return userDtoMapper.parse(userService.get(userId));
```

I don't think we need to catch here this exception and re-throw RuntimeException